CMakeLists.txt

```diff
@@ -8,10 +8,26 @@ set(SOURCE_FILES ${SOURCE_FILES} include/graphics.c)
 
 add_executable(${PROJECT_NAME} ${SOURCE_FILES})
 
+if(${CMAKE_HOST_APPLE})
+    set(SDL2_TTF_INCLUDE_DIRS "/usr/local/Cellar/sdl2_ttf/2.0.15/include/SDL2")
+    set(SDL2_TTF_LIBRARIES "-L/usr/local/Cellar/sdl2_ttf/2.0.15/lib" "-lSDL2_TTF")
+else()
 INCLUDE(FindPkgConfig)
 
 pkg_check_modules(SDL2 REQUIRED sdl2)
 pkg_check_modules(SDL2_TTF REQUIRED SDL2_ttf)
+endif()
```

Comment:
SDL2 it self was working as expected, only TTF was having heavy issues. I will test this after finishing the review.